README.md

```diff
@@ -130,8 +130,8 @@ cluster:
             apiAudiences:
               templateName: awsApiServerApiAudiences # name of the Helm template that renders api-audiences value
             featureGates:
-              - enabled: true
-                name: CronJobTimeZone
+            - name: StatefulSetAutoDeletePVC
+              enabled: true
```

Comment:
Why do we introduce a new feature gate here?
If it's intended I'd say we should mention it in the changelog.

Reply:
This is just in the `README.md` as an example of how customers can modify their cluster.